symfony/plugins/orangehrmCorePlugin/Dao/MenuDao.php

```diff
@@ -118,4 +119,32 @@ public function enableModuleMenuItems(string $moduleName, array $menuTitles = []
             throw new DaoException($e->getMessage(), $e->getCode(), $e);
         }
     }
+
+    /**
+     * @param string $screenName
+     * @return MenuItem|null
+     */
+    public function getMenuLevel(string $screenName): ?MenuItem
```

Comment:
```php
public function getMenuLevel(string $module, string $screenName)
```
and method name should change. e.g. `getMenuItemByModuleAndScreen`

Reply:
done